### DISCUSSION TITLE
shell completion for additional sources in searchpath?

### DISCUSSION THREAD

**Discussion by brijow** [facebookresearch/hydra#2421] [Q&A] [answered]
I was following this section of the docs that discuss the config search path: https://hydra.cc/docs/advanced/search_path/, but when trying out the example here https://github.com/facebookresearch/hydra/tree/main/examples/advanced/config_search_path I was unable to access the imagenet dataset name via tab completion: 

```{bash}
python my_app.py dataset=i   # hitting tab here does not work (after installing shell completion)
```

The example itself works fine, that is , if I manually type  out `dataset=imagenet` I see the expected config composed from that additional source.

I was wondering if there is a way to specify these additional config sources and have shell completion working for those?

**Comment by Jasha10** [comment, marked as answer]
This should be addressed by PR #2426.
